nin.json

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "My project",
+  "authors": [
+    "Your demoscene handle"
+  ],
+  "description": "This is my project",
+  "previewImage": "https://raw.githubusercontent.com/ninjadev/nin/master/nin/frontend/app/images/nin-dark.png",
+  "music": {
+    "path": "res/music.mp3",
+    "bpm": 125,
+    "subdivision": 4
+  },
+  "googleAnalyticsID": "Your GA ID"
+}
```

Comment:
Yeah, when generating json files you need to manually add a `\n` to the end. Easy to forget, should be fixed in nin. Please create an issue for it.